config/locales/en.yml

```diff
@@ -225,6 +225,9 @@ en:
       transparency:
         one: Transparency data
         other: Transparency data
+      world_location_news_article:
```

Comment:
@Rosa-Fox - what did content think about simply leaving out the `format` from the history notice? That would mean everything would simply say "This was published under the..."?

Instead of a condition in the view, we could also use Rails' [`default`](http://guides.rubyonrails.org/i18n.html#defaults) option in the `I18n.t` - so we could have something like

```ruby
I18n.t(
       "content_item.format.#{content_item.format}",
       default: ["content_item.format.#{content_item.document_type}".to_sym],
       count: 1
      ).downcase
```

Then if there were no translations for "World location news article" it would fall back to using `document_type` which would be "news article". My only real concern here is that it isn't really a default, it's a fallback so the naming is annoying.